storage/signed_urls/generate_signed_urls.py

```diff
@@ -105,7 +108,7 @@ def generate_signed_url(service_account_file, bucket_name, object_name,
     for k, v in ordered_query_parameters.items():
```

Comment:
Variable names. Something like safe_name, safe_value would be clearer than encoded_k and encoded_v